roles/splunk_monitor/defaults/main.yml

```diff
@@ -0,0 +1,2 @@
+---
+# defaults file for splunk_monitor
```

Comment:
I can't remember if this is true but the ansible linting might complain about the lack of newline here.

Reply:
nevermind, ci came back green